fvm/evm/handler/addressAllocator.go

```diff
@@ -40,6 +40,10 @@ func (aa *AddressAllocator) COAFactoryAddress() types.Address {
 	return MakeCOAAddress(0)
 }
 
+func (aa *AddressAllocator) NativeTokenBridgeAddress() types.Address {
```

Comment:
can you add a comment explaining why we need this account (for nonce tracking etc)